framework/include/cppmicroservices/AnyMap.h

```diff
@@ -364,7 +385,42 @@ namespace cppmicroservices
     class US_Framework_EXPORT AnyMap : public any_map
     {
       public:
-        AnyMap(map_type type = any_map::UNORDERED_MAP_CASEINSENSITIVE_KEYS);
+        /**
+         * @brief initializer_list constructor
+         *
+         * Construct an AnyMap of type UNORDERED_MAP_CASEINSENSITIVE_KEYS with the content of the
+         * initialized with the content of the the initializer list. Allows for inline
+         * initialization akin to:
+         *
+         *     AnyMap cache_bundle1 {
+         *         {"a", std::string("A")},
+         *         {"b", std::string("B")},
+         *         {"c", std::string("C")}
+         *     };
+         * @param l a std::initializer_list<value_type> used to initialize the content of the AnyMap.
+         *
+         */
+        AnyMap(std::initializer_list<any_map::value_type> l = {});
+        /**
+         * @brief initializer_list constructor
+         *
+         * Construct an AnyMap of type "type" with the content of the
+         * initialized with the content of the the initializer list. Allows for inline
```

Comment:
this sentence doesn't read quite right; `with the content of the initialized with the content of the the initializer list.`

Reply:
Fixed.